test/guest/cockpit-fedora-23.setup

```diff
@@ -17,7 +17,7 @@ echo 'SELINUX=permissive' > /etc/selinux/config
 # them to not spontaneously change from one test run to the next when
 # the distribution repository is updated.
 #
-COCKPIT_DEPS="json-glib realmd glib-networking libssh selinux-policy-targeted docker pcp-libs pcp kubernetes etcd subscription-manager storaged storaged-lvm2 device-mapper-multipath"
+COCKPIT_DEPS="json-glib realmd glib-networking libssh selinux-policy-targeted docker pcp-libs pcp kubernetes etcd subscription-manager storaged storaged-lvm2 device-mapper-multipath atomic"
```

Comment:
Can we multiline these deps [like this](https://github.com/cockpit-project/cockpit/blob/64be41182486f0180c07c93946fed1a2f16bcb45/test/guest/cockpit-rhel-7.setup#L47)? It makes things more readable, especially if the dependencies change.